relayer/pkg/chainlink/ocr2/medianreport/report.go

```diff
@@ -52,6 +52,8 @@ func (c ReportCodec) BuildReport(oo []median.ParsedAttributedObservation) (types
 	juelsPerFeeCoin = starknet.SignedBigToFelt(juelsPerFeeCoin) // converts negative bigInts to corresponding felt in bigInt form
 	juelsPerFeeCoinFelt := junotypes.BigToFelt(juelsPerFeeCoin)
 
+	// TODO: source and include gasPrice
```

Comment:
Note: a value needs to be added here, then the aggregator contracts package needs to be rebuilt and uploaded to npm. Tests currently pass since it's still using the old contract versions